python/createMultipleLmdData.py

```diff
@@ -180,7 +180,7 @@
     job = Job(
         resource_request,
         f"{lmdScriptPath}/singularityJob.sh {lmdScriptPath}/createLumiFitData.sh",
-        "createLumiFitData",
```

Comment:
we shouldn't hardcode this at all, there needs to be a reference to the config here